base_installation.sh

```diff
@@ -36,12 +36,7 @@ pip=`which pip`
 if [ -e ~/.local/bin/pip ] ; then
   pip=~/.local/bin/pip
 fi
-sudo $pip install --user numpy --ignore-installed
-sudo $pip install --user celery --ignore-installed
-sudo $pip install --user taxtastic --ignore-installed
-sudo $pip install --user lxml --ignore-installed
-sudo $pip install --user python-gflags --ignore-installed
-sudo $pip install --user ete2 --ignore-installed
+sudo $pip install --user -r requirements.txt --ignore-installed
```

Comment:
The sudo is not needed. as noted in the last line of the https://github.com/Metannotate/Metannotate/issues/5. The normal reason you use ```--user``` is so you don't need to to use ```sudo```.